src/modules/shared/pages/innovation/documents/documents-list.component.ts

```diff
@@ -206,7 +214,11 @@ export class PageInnovationDocumentsListComponent extends CoreComponent implemen
     this.onFormChange();
   }
 
-  calculateFilterNum() {
+  onSearchClick() {
+    this.form.get('name')?.updateValueAndValidity({ onlySelf: true });
```

Comment:
Just wondering why here it's `.get('name')?` but on line 124 it's `.get('name')!`, I guess we feel less confident about its existence by the time we get to line 218 😆